apache.yml

```diff
@@ -12,46 +12,27 @@ resource:
       tags:
         Name: '${var.koding_user_username}-${var.koding_group_slug}'
       user_data: |-
-        set -e
+        #!/bin/bash
+
+        set -xe
```

Comment:
I'm okay with it. I just don't make any exceptions for option names in shell scripts :)